internal/configs/transportserver_test.go

```diff
@@ -366,3 +365,89 @@ func TestGenerateUnixSocket(t *testing.T) {
 		t.Errorf("generateUnixSocket() returned %q but expected %q", result, expected)
 	}
 }
+
+func TestGenerateTransportServerHealthChecks(t *testing.T) {
+	tests := []struct {
+		healthCheck *conf_v1alpha1.HealthCheck
+		expected    *version2.StreamHealthCheck
+	}{
```

Comment:
I suggest adding the msg field to describe each test case, as it is not immediately clear what test cases we test here